subsys/bluetooth/controller/hal/nrf5/radio/radio.c

```diff
@@ -109,9 +109,9 @@ void radio_phy_set(u8_t phy, u8_t flags)
 #endif /* CONFIG_BT_CTLR_RADIO_ENABLE_FAST */
 }
 
-void radio_tx_power_set(u32_t power)
+void radio_tx_power_set(s8_t power)
```

Comment:
Wasn't there a reason to use `u32_t` here? I seem to recall I changed this a while back

Reply:
I just go inspired by the nrfx implementation... but I prefer u32_t... but but... if a host API is implementation I would assume it would pass a s8_t  downstream.

Reply:
FWIW, NRF_RADIO_Type for NRF52 has the following definition:

```C
  __IO uint32_t  TXPOWER;                           /*!< Output power
```

Reply:
Ok, will revert back to u32_t as was my early code.